src/libs/actions/Report.js

```diff
@@ -196,12 +196,12 @@ function getSimplifiedReportObject(report) {
  */
 function getSimplifiedIOUReport(reportData, chatReportID) {
     const transactions = _.map(reportData.transactionList, transaction => ({
-        transactionID: transaction.transactionID,
+        transactionID: Number(transaction.transactionID),
```

Comment:
Will add a comment.

Reply:
I don't think the comment answers "why". In fact, I don't think it CAN be cast to a number and should remain as a string. This is because many transactionIDs are too large of a number for JS to handle and will end up with math rounding errors when cast to a number (eg. ...12345 can become ...12346)

Reply:
Unresolving, since otherwise this is easy to miss

Reply:
Thanks, I missed this one. 

Now ensuring that that the `transactionID` is resolved as a String. I'll most likely be able to remove the iou transactions entirely in the 'cancel/decline transactions' IOU issue 🤞

Reply:
For comparison, I'm converting the number to a String, this seemed better than ignoring the lint error for using `==`.